csfieldguide/templates/chapters/index.html

```diff
@@ -13,15 +13,25 @@ <h1 id="chapters" class="section-heading anchor-link">Chapters</h1>
 
 {% block content %}
 
-  {% if all_chapters %}
-      {% for chapter in all_chapters %}
-        <a href="{% url 'chapters:chapter' chapter.slug %}">
-          <img src="{% if chapter.icon %}{% static chapter.icon %}{% else %}http://placehold.it/238x200{% endif %}">
-          <h3>{{ chapter.name }}</h3>
-        </a>
-      {% endfor %}
-  {% else %}
-    <p>No chapters are available.</p>
-  {% endif %}
+  <div class="row chapter-list">
+    
+      {% if all_chapters %}
+          {% for chapter in all_chapters %}
+          <div class="l12 chapter-list-item">
+            <div class="row">
```

Comment:
It would be nice if all this content is center aligned when viewing on a mobile device.